man/systemd-journal-gatewayd.service.xml

```diff
@@ -58,26 +58,27 @@
       <varlistentry>
         <term><option>--cert=</option></term>
 
-        <listitem><para>Specify the path to a file containing a server
-        certificate in PEM format. This option switches
-        <command>systemd-journal-gatewayd</command> into HTTPS mode
-        and must be used together with
-        <option>--key=</option>.</para></listitem>
+        <listitem><para>Specify the path to a file containing a server certificate in PEM format. This option
+        switches <command>systemd-journal-gatewayd</command> into HTTPS mode and must be used together with
+        <option>--key=</option>. If the path refers to an <constant>AF_UNIX</constant> stream socket in the
+        file system a connection is made to it and the certificate read from it.</para></listitem>
```

Comment:
That seems roundabout. Maybe just say:

`A path to a file or <constant>AF_UNIX</constant> stream socket to read the server certificate from. The certificate must be in PEM format. This option switches ...`

(The proposed version talks about the argument and the format in the first sentence, then about journald, then about the argument again.)